train.py

```diff
@@ -83,6 +85,9 @@ def process_args():
     argparser.add_argument('--resume-subdir',
                         default=None,
                         help='Subdirectory containing architecture json and weights')
+    argparser.add_argument('--pickle',
+                        default=None,
+                        help='Name of folder with pickled dataframes')
```

Comment:
It might be nice to merge some basic pickles into the repository so that everyone has the same set of `train/val/test` to work with. Obviously people could create new ones but a basic set to get started is nice to quickly clone a new repo and start training without needing to download anything else. How big are these files btw? (not sure if we wanna avoid checking big files into the repo)

Reply:
Full set of Train/Val/Test with no scaling is 151MB

Reply:
We could deploy them via GitHub Package Releases. Up to 1 GB per file, and I believe it also generates a link we can use in our scripts. What was the downside to doing the splitting in-memory? Was it so we have consistent train/val splits on the training set (i.e. no chance of a different split across training sessions)? Could we just save a csv of annotation IDs, and populate the data frames using those IDs?

Reply:
Consistency between the sets. If we get ejected from colab deep into a training set, having the splits saved would allow training to resume with identical training/validation/testing as before. If we wanted to train for multiple days across the same dataset we no longer need to worry about the 24hr timeout. 

Saving it as a Csv is doable, but It’s [much slower than pickle](https://towardsdatascience.com/the-best-format-to-save-pandas-data-414dca023e0d?gi=c431087feed5)

I’ll look into GitHub package releases tomorrow.

Reply:
Sounds good, thanks for the clarification. Fully agree with your analysis. Then I think GitHub package releases might be the best option for us. We would be limited to 2 GB per 'release' which sounds like it's plenty of breathing room for us. We'd need to confirm that the package URL stays constant.